priv/templates/boot.eex

```diff
@@ -208,6 +208,15 @@ _replace_os_vars() {
         }1' < "$1" > "$2"
 }
 
+# Private. Run hooks from directory.
+_run_hooks_from_dir() {
+   if [ -d "$1" ]; then
+          for s in $(find "$1" -name "*.sh" -print | sort); do
```

Comment:
That's a good point, my grep is erroneous. The `find ... -print0 | while read -d $'\0' s ; do`I think is safe and sane for now.